health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -447,28 +449,38 @@
     }
     setSubmitting(true);
     try {
-      const data = await mutateAsync(payload); // Use mutateAsync for await support
-      // Handle successful checklist creation  
-      // Proceed with localization if needed
+      // Prepare localization data
       let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
       if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
       let roleTemp = role.toUpperCase().replace(/ /g, "_");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)